.github/workflows/iroha2-custom-image.yml

```
@@ -1,38 +1,20 @@
 name: I2::Custom::Publish
 
 on:
-  push:
-    tags:
-      - 'v2*'
   workflow_dispatch:
     inputs:
-      BUILD_GLIBC_IMAGE:
-        description: "Select \"true\" to build and push the standard \"glibc\" image"
-        type: choice
-        required: true
-        default: 'false'
-        options:
-          - true
-          - false
-      BUILD_ALPINE_IMAGE:
-        description: "Select \"true\" to build and push the alpine image from specific commit SHA"
-        type: choice
+      PROFILER_NAME:
+        description: "Profiler value (profiling/glibc/alpine) as the image tag part"
         required: true
-        default: 'false'
-        options:
-          - true
-          - false
+        type: string
```

If we want to go this way,

```suggestion
        type: choice
        options:
          - profiling
          - glibc
          - alpine
```